packages/@ember/-internals/metal/lib/computed.ts

```diff
@@ -804,8 +930,44 @@ class ComputedDecoratorImpl extends Function {
   client.get('fullName'); // 'Betty Fuller'
   ```
 
-  You can pass a hash with two functions, `get` and `set`, as an
-  argument to provide both a getter and setter:
+  You can also provide a setter, either directly on the class using native class
+  syntax, or by passing a hash with `get` and `set` functions.
+
+  Example:
+
+  ```js
+  import { computed, set } from '@ember/object';
+
+  class Person {
+    constructor() {
+      this.firstName = 'Betty';
+      this.lastName = 'Jones';
+    },
+
+    @computed('firstName', 'lastName')
+    get fullName() {
+      return `${this.firstName} ${this.lastName}`;
+    }
+
+    set fullName(value) {
+      let [firstName, lastName] = value.split(/\s+/);
+
+      set(this, 'firstName', firstName);
+      set(this, 'lastName', lastName);
+
+      return value;
+    }
+  }
```

Comment:
<3 Thanks so much for adding this (as well as the rest of these)! Makes a big difference.